websrc/default_hands/src/default_hands_main.tsx

```diff
@@ -960,6 +1017,16 @@ class DefaultHands extends React.Component< {}, {} >
 			}
 		};
 
+		if (this.state.menuGestureCollideMain && this.state.menuGestureCollideSecondary && Date.now() >= this.menuGestureCooldownCounter + 500)
+		{
+			this.menuGestureCooldownCounter = Date.now();
+			this.toggleGadgetMenu();
+			this.displayIntro = false;
+		}
```

Comment:
Render seems like the wrong place to trigger an action like this. Maybe componentDidUpdate would be better?

I would also move displayInto into state so it triggers a re-render.